lib/galaxy/datatypes/test/2.txt

```diff
@@ -1 +1 @@
-a	2
\ No newline at end of file
+a	2
```

Comment:
Is this changing the newline used? Why?

Reply:
Yes, so that we can verify that a upload-> download cycle still results in the same file (using a hash). We could also allow some diff, or skip that test.

Reply:
Or test without newline conversion